packages/gamut-labs/src/AppHeader/Search/SearchButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { SearchIcon } from '@codecademy/gamut-icons';
+import { system } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+const SearchIconButton = styled.button(
+  system.css({
+    backgroundColor: 'transparent',
+    pt: 8,
+    border: 'transparent',
+    '&:hover': {
+      color: 'hyper',
+      cursor: 'pointer',
+    },
+    '&:focus': {
+      outline: 'none',
+    },
+    '&:focus-visible': {
+      outline: 'theme.colors.navy',
```

Comment:
i think this is your outline problem -- i think you can just pass `navy` here 🙂

```suggestion
      outline: 'navy',
```

Reply:
oooooh then this is a Gamut typings issue in that it's allowed??